NEWS.md

```diff
@@ -1,20 +1,18 @@
-shiny 1.3.1
+shiny 1.3.2
```

Comment:
I think we should use 1.3.1.9000, since it's possible there will be more changes. Also, update the DESCRIPTION?